src/microbe_stage/components/MicrobeControl.cs

```diff
@@ -181,6 +182,10 @@ public static void SetMoveSpeed(this ref MicrobeControl control, float speed)
         if (organelleInfo.SlimeJets == null || organelleInfo.SlimeJets.Count < 1)
             return;
 
+        // Not valid if all slime jets are mucocysts
+        if (organelleInfo.SlimeJets.All(c => c.IsMucocyst))
```

Comment:
The slimejet detection system should instead be made to not consider mucocysts slime jets, as they are not slime jets at all.

Reply:
Isn't it doing similiar (if all slime jets are mucocyst then return, 'cause all slime jets are not normal slime jets)?

Reply:
Why are mucocysts placed in the list that is *only for slimejets*? That doesn't make sense to me and makes the whole code base design much more complex when the list may or may not contain something that doesn't match the variable name.